crates/test-files/fixtures/compile_errors/missing_return_in_else.fe

```diff
@@ -3,4 +3,4 @@ contract Foo:
         if val > 1:
             return 5
         else:
-            x = 1
```

Comment:
This was an extra error in the test file that I think we didn't intent to test for here